Sources/NIO/Utilities.swift

```diff
@@ -41,7 +52,34 @@ public enum System {
     ///
     /// - returns: The logical core count on the system.
     public static var coreCount: Int {
+#if os(Windows)
+        var dwLength: DWORD = 0
+        _ = GetLogicalProcessorInformation(nil, &dwLength)
+
+        let alignment: Int =
+            MemoryLayout<SYSTEM_LOGICAL_PROCESSOR_INFORMATION>.alignment
+        let pBuffer: UnsafeMutableRawPointer =
```

Comment:
aren't we missing a `defer { ...deallocate() }` here?

Reply:
Yeah, this is missing the defer ... thats what I get for trying to push this into a separate change (I have that locally).